controllers/auth.js

```diff
@@ -0,0 +1,58 @@
+const User = require('../models/User')
+const bcrypt = require('bcrypt')
+
+const create = async ( req, res ) => {
+    try {
+      
+      let {password} = req.body
+      const {email, avatar, likedEvents, dislikedEvents, blacklistEvents} = req.body
+      const duplicateUser = await User.findOne({email})
+
+      //test for duplicate users
+      if(duplicateUser) {
+          return res.status().json({
+              status: 400,
+              duplicateUser,
```

Comment:
I don't think we need to send back the duplicate user data. Otherwise someone just enters a random email (that don't belong to them) in the front end, and they would've gotten access to the matching user's data back...